blueprints/app/files/package.json

```diff
@@ -63,6 +64,7 @@
     "ember-cli-babel": "^8.2.0",
     "ember-cli-clean-css": "^3.0.0",
     "ember-cli-dependency-checker": "^3.3.2",
+    "ember-cli-deprecation-workflow": "^3.0.2",
```

Comment:
this would need to be 3.1.0 now to have types available

Reply:
updated!